_posts/en/newsletters/2020-09-09-newsletter.md

```diff
@@ -0,0 +1,196 @@
+  Fully addressing the attacks might also require some other changes
+  related to timelocks, such as using a 1-block relative timelock ("`1
+  OP_CSV`") to prevent pinning.
+
+  In addition to the discussion helping to improve a privacy-enhancing
+  coinswap protocol, at least one participant [noted][zmnscpxj ln] that
+  some of the insights from the conversation might be applicable to
+  other protocols, such as LN.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Re-delegate absurd fee checking from mempool to clients][review club #19339] is
+a PR ([#19339][Bitcoin Core #19339]) by Gloria Zhao that proposes an updated
+version of [Bitcoin Core #15810][]. It adds separate fee-checking logic for
+clients that need to enforce max feerates and then removes the `nAbsurdFee`
+logic from `AcceptToMemoryPool`.
+
+The review club discussion covered general concepts before diving into the
+deeper technical aspects.
+
+{% include functions/details-list.md
+  q0="What does the `AcceptToMemoryPool` (ATMP) function in `validation.cpp` do?"
+  a0="ATMP is the function that Bitcoin Core calls to determine whether a transaction
+      is acceptable for inclusion in the mempool. In addition to
+      enforcing consensus rules, ATMP also applies the node's policy."
+  a0link="https://bitcoincore.reviews/19339#l-55"
+
+  q1="What is \"policy\"?"
+  a1="Policy is generally defined as each node's network transaction rules.  It excludes
+      consensus rules and the preferences of end-user wallets."
+  a1link="https://bitcoincore.reviews/19339#l-281"
+
+  q2="What is the absurd fee parameter, `nAbsurdFee`, used for?"
+  a2="`nAbsurdFee` is only used for locally submitted transactions, e.g. those
+      originating from the node's RPC interface and wallet. When a node receives
+      a transaction from the network (which is the majority of calls to ATMP),
+      it sends an empty value for `nAbsurdFee`."
+  a2link="https://bitcoincore.reviews/19339#l-85"
+
+  q3="Is `nAbsurdFee` part of policy, or is it a wallet preference?"
+  a3="The participants concluded that `nAbsurdfFee` is a wallet preference and
+      not policy, as it is not enforced on the P2P network by Bitcoin Core and
+      [ATMP should give consistent results][review club client] regardless of
+      which client the transaction originates from."
+  a3link="https://bitcoincore.reviews/19339#l-91"
+
+  q4="Why does the `testmempoolaccept` RPC accept a JSON array containing a
+      maximum of only one raw transaction?"
+  a4="It's hoped that Bitcoin Core will one day be able to test *multiple*
+      transactions in a single RPC without changing the interface, if package
+      relay and package mempool acceptance are implemented."
```

Comment:
Oh, I forgot to include possible Optech topic links, like here for package relay. Not sure if they are good to add in the quiz.